vllm/model_executor/weight_utils.py

```diff
@@ -250,6 +262,20 @@ def hf_model_weights_iterator(
             with open(param_path, "rb") as f:
                 param = np.load(f)
             yield name, torch.from_numpy(param)
+    elif load_format == "tensorizer":
+        logger.warning("Deserializing HuggingFace models are not optimized for "
```

Comment:
pretty sure "are not optimized" should be "is not optimized". The subject of the verb here is "(tensorizer) deserializing"